ambry-clustermap/src/main/java/com.github.ambry.clustermap/StaticParticipantFactory.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.clustermap;
+
+import com.github.ambry.config.ClusterMapConfig;
+import com.github.ambry.config.VerifiableProperties;
+
+
+/**
+ * A no-op {@link ClusterParticipant} associated with static cluster management.
+ */
+class StaticParticipantFactory implements ClusterParticipantFactory {
```

Comment:
Just so that the `AmbryServer` could instantiate either the static or the dynamic cluster manager based on just the config parameter.